docs/user-guide/petset.md

```diff
@@ -161,7 +199,7 @@ pod "web-0" deleted
 pod "web-1" deleted
 ```
 
-Wait for them to come back up, and try to retrieve the previously written hostname through the DNS name of the peer (remember the Pet Set also gives us stable storage, and that the hostname is linked to the DNS name).
+Wait for them to come back up, and try to retrieve the previously written hostname through the DNS name of the peer. They match because the storage, DNS name and hostname stick to the Pet no matter where it gets scheduled:
```

Comment:
They match, because the storage, DNS name, and hostname stick to the Pet no matter where it gets scheduled: